components/metadata-service/internal/metadata/servicedefservice_test.go

```diff
@@ -660,11 +718,26 @@ func TestServiceDefinitionService_GetAll(t *testing.T) {
 
 		// when
 		result, err := service.GetAll("re")
-		require.NoError(t, err)
 
 		// then
+		require.NoError(t, err)
 		assert.Len(t, result, 0)
 	})
+
+	t.Run("should return 404 if cannot find Remote Environment", func(t *testing.T) {
```

Comment:
`t.Run("should return not found error if cannot find Remote Environment", func(t *testing.T) {`